tests/test_fft.py

```diff
@@ -1,79 +1,52 @@
 from unittest import TestCase
 import numpy as np
+
 from aspire import config
 from aspire.utils.numeric import fft_class
 
-fft_backends = ('scipy', 'pyfftw')
+fft_backends = ['scipy', 'pyfftw']
+numeric_classes = [np, np]
+
+# Create Cupy fft backend if Cupy is enabled and lib exits.
+if config.common.cupy:
```

Comment:
so I don't think this check is necessary. After all, we're ignoring `config.ini` for these tests when it comes of selection of the fft backend, so why not ignore what it says about `cupy` too? If we're able to import cupy, we might as well test it.

Reply:
It failed when I run it on head node of tiger cluster where I installed the cupy but the GPU device is not available. config.common.cupy is kind of for users to make sure to run cupy test with GPU device.

Reply:
ok..no problem